src/Templates/src/templates/maui-mobile/AppShell.xaml

```diff
@@ -4,7 +4,7 @@
     xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
     xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
     xmlns:local="clr-namespace:MauiApp._1"
-    Shell.FlyoutBehavior="Disabled"
+    Shell.FlyoutBehavior="Flyout"
```

Comment:
I do think this change is somewhat correct

I think we should just delete Shell.FlyoutBehavior="Disabled" from the template

Having it disabled is somewhat confusing.
It's already "disabled" by default because it's only a single page. It only shows up if users add FlyoutItems. 

It's currently kind of confusing when you add FlyoutItems and nothing shows up